test/README.md

```diff
@@ -1,3 +1,71 @@
 # Tests
 
-For more info on the tests see [Eventing Tests](https://github.com/knative/eventing/blob/main/test/README.md)
+General Knative Eventing testing topics covered in [Knative Eventing Tests](https://github.com/knative/eventing/blob/main/test/README.md)
+
+## Testing Eventing RabbitMQ
+
+Eventing RabbitMQ testing workflow is centered around Makefile.
+Things like initializing and setting-up kind cluster, installing
+components and running tests - everything done with `make`.
+
+### Create environment
+
+```
+make .envrc && . .envrc
+```
+
+Here we create our environment file containing patched PATH, various exports, etc.
+I personally like to use [direnv](https://direnv.net/) too. It will pick up .envrc and source it
+each time I cd'ing to my Eventng RabbitMQ local copy and more...
+
+### Create kind cluster
+
+```
+make kind-cluster
+```
+
+This will install `kind` to ./bin and initialize the cluster. Kind uses [kind.yaml](e2e/kind.yaml) as cluster configuration.
```

Comment:
```suggestion
This will install `kind` to the `./bin` directory and initialize the cluster. Kind uses [kind.yaml](e2e/kind.yaml) as cluster configuration.
```